packages/components/molecules/f-breadcrumbs/src/components/Breadcrumbs.vue

```diff
@@ -1,46 +1,119 @@
 <template>
     <div
-        :class="$style['c-breadcrumbs']"
-        data-test-id="breadcrumbs-component">
-        {{ copy.text }}
+        data-test-id="breadcrumbs-component"
+        :class="$style['c-breadcrumbs']">
+        <ul :class="$style['c-breadcrumbs-list']">
+            <template v-for="({ name, url }, index) in links">
+                <li
+                    v-if="index !== 0"
+                    :key="`${index}_link`"
+                    :class="$style['c-breadcrumbs-separator']">
+                    >
+                </li>
+                <li
+                    :key="`${index}_separator`"
+                    :class="$style['c-breadcrumbs-item']">
+                    <router-link
+                        v-if="routerLinks"
+                        :to="url"
+                        :class="[
+                            $style['c-breadcrumbs-link'],
+                            linkActiveClass(index)
+                        ]">
+                        {{ name }}
+                    </router-link>
+                    <a
+                        v-else
+                        :href="url"
+                        :class="[
+                            $style['c-breadcrumbs-link'],
+                            linkActiveClass(index)
+                        ]">
+                        {{ name }}
+                    </a>
+                </li>
+            </template>
+        </ul>
     </div>
 </template>
 
 <script>
-import { globalisationServices } from '@justeat/f-services';
-import tenantConfigs from '../tenants';
 
 export default {
     name: 'Breadcrumbs',
-    components: {},
     props: {
-        locale: {
-            type: String,
-            default: ''
+        links: {
+            type: Array,
+            default: () => []
+        },
+        routerLinks: {
+            type: Boolean,
+            default: true
```

Comment:
On by default seems to be a bad choice to me here - you might not be using router everywhere. Additionally, there might be some links that are `router-link`able, while others are on a different microsite - would it be possible to have this in as a per-link setting in the links array?

Reply:
Yea I never thought about the microsite point, yep happy to make it on a per link basis